cirq/qis/states_test.py

```diff
@@ -36,46 +37,69 @@ def assert_valid_density_matrix(matrix, num_qubits=None, qid_shape=None):
                                      dtype=matrix.dtype), matrix)
 
 
-def test_bloch_vector_simple_H_zero():
-    sqrt = np.sqrt(0.5)
-    H_state = np.array([sqrt, sqrt])
+@pytest.mark.parametrize('global_phase', (1, 1j, np.exp(1j)))
+def test_bloch_vector_zero_state(global_phase):
+    zero_state = global_phase * np.array([1, 0])
 
-    bloch = cirq.bloch_vector_from_state_vector(H_state, 0)
-    desired_simple = np.array([1, 0, 0])
+    bloch = cirq.bloch_vector_from_state_vector(zero_state, 0)
+    desired_simple = np.array([0, 0, 1])
     np.testing.assert_array_almost_equal(bloch, desired_simple)
 
 
-def test_bloch_vector_simple_XH_zero():
+@pytest.mark.parametrize('global_phase', (1, 1j, np.exp(1j)))
+def test_bloch_vector_one_state(global_phase):
+    zero_state = global_phase * np.array([0, 1])
```

Comment:
This says test_bloch_vector_**one_state** but then has zero_state.  Is this a typo?

Reply:
typo. fixed.